pkg/provider/webhook/webhook_test.go

```diff
@@ -220,6 +220,19 @@ want:
   err: ''
   result: secret-value
 ---
+case: good json with jsonpath filter
+args:
+  url: /api/getsecret?id={{ .remoteRef.key }}&version={{ .remoteRef.version }}
+  key: testkey
+  version: 1
+  jsonpath: $.secrets[?@.name=="thesecret"].value
+  response: '{"secrets": [{"name": "thesecret", "value": "secret-value"}, {"name": "alsosecret", "value": "another-value"}]}'
+>>>>>>> 2c2047c4 (Add jsonpath filter support to webhook)
```

Comment:
theres a marker from the merge conflict here

Reply:
oups sorry, I missed it